backend/src/model/dto.rs

```diff
@@ -216,6 +217,20 @@ pub struct NewMapDto {
     pub location: Option<Coordinates>,
 }
 
+/// The information for updating a map.
+#[typeshare]
+#[derive(Serialize, Deserialize, ToSchema)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Deserialize, ToSchema)]
```

Reply:
Serialize is needed for the tests, since we want to append an UpdateMapDto to the test request.